influxdb/client_test.go

```diff
@@ -196,6 +197,59 @@ func TestClient_Write(t *testing.T) {
 	}
 }
 
+func TestClient_WriteLarge(t *testing.T) {
+	expected := &bytes.Buffer{}
```

Comment:
This writes a million points to an httptest server and checks to make sure that it is delivered properly.  The million points are gzipped.